physics/PBL/SATMEDMF/satmedmfvdifq.F

```diff
@@ -97,6 +97,7 @@ subroutine satmedmfvdifq_run(im,km,ntrac,ntcw,ntrw,               &
      &                        ntke, ntqv
       integer, intent(in)  :: sfc_rlm
       integer, intent(in)  :: tc_pbl
+      integer, intent(in)  :: use_lpt
```

Comment:
Can this be a logical instead of an integer? Will it ever have a value other than 0 or 1?